runtime/src/chpl-file-utils.c

```diff
@@ -55,6 +56,27 @@ qioerr chpl_fs_chown(const char* name, int uid, int gid) {
   return err;
 }
 
+qioerr chpl_fs_copy_metadata(const char* source, const char* dest) {
+  qioerr err = 0;
+  struct stat oldTimes;
+  struct utimbuf times;
+  int exitStatus = stat(source, &oldTimes);
+  if (exitStatus == -1) {
+    // Hopefully an error will not occur (as we have checked that the
+    // file exists when we perform the other operations on it).  But just in
+    // case, check it here.
+    err = qio_mkerror_errno();
+    return err;
+  }
+  times.actime = oldTimes.st_atime;  // The access time
+  times.modtime = oldTimes.st_mtime; // The modification time
+  exitStatus = utime(dest, &times);  // Set the times for dest.
+  if (exitStatus == -1) {
+    err = qio_mkerror_errno();
+  }
+  return err;
+}
+
```

Comment:
I'm a little suprised that fs_copy_metadata does not copy the mode, user id, or group id. That's OK, but you might put a comment about that...